gosling/display.py

```diff
@@ -48,7 +48,7 @@ class GoslingDeps:
 
 def get_display_dependencies(
     gosling_version: str = SCHEMA_VERSION.lstrip("v"),
-    higlass_version: str = "1.11",
+    higlass_version: str = "1.12",
```

Comment:
I was able to replicate the brush error on my local machine, and I think it is because the higlass version here is `1.12` which we did not yet support in Gosling.js (i.e., gosling.js v0.9.28 uses HiGlass@1.11.8).

I think the error should be due to the recent major version upgrade of `d3-selection` on HiGlass, and I will need to first support newest HiGlass on Gosling.

Reply:
(I can confirm that I confronted the same brush issue when I upgrade HiGlass in Gosling.js)

Reply:
hmm ok, I couldn't get v1.11 to work from esm.sh. I'll try again later

Reply:
If gosling isn't compatible with higlass v1.12, then we should pin 1.11 in the package.json

Reply:
We already pinned the version under [`dependency` of `package.json`](https://github.com/gosling-lang/gosling.js/blob/a107df73a9934e47ead392937e2e8b2629dd2bfd/package.json#LL72C30-L72C30). I meant to support higlass v1.12, but it is a quite recent release, and I did not find time to upgrade it. I will work on it soon though.

Btw, do we need to provide a higlass version here at all? Gosling includes higlass as a dependency (not a peer dependency).

Reply:
Sorry, I should be more clear.

We pin `^1.11.8` in the `package.json`, which is **not an exact version**. The `^` carries semantics that it's OK to bump up to a **minor** version while is _not_ ok to bump to the next **major** verison. The `^` and `~` specify what semver range of the package [are compatible](https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json). You can check and see how `^1.11.8` is resolved by following the unpkg URL https://unpkg.com/higlass@^1.11.8.

If we wanted to say "only higlass v1.11.8" then our package.json, then we should have:

```
  "higlass": "1.11.8"
```

However, it is probably preferable to use the `~` so that we can get any patch fixes for HiGlass.


```diff
-- "higlass": "^1.11.8"
++ "higlass": "~1.11.8"
```